bin/millau/node/src/chain_spec.rs

```diff
@@ -164,3 +165,12 @@ fn testnet_genesis(
 		}),
 	}
 }
+
+fn load_rialto_bridge_config() -> Option<BridgeRialtoConfig> {
+	Some(BridgeRialtoConfig {
+		initial_header: Some(millau_runtime::rialto::initial_header()),
```

Comment:
It should be `rialto_runtime::initial_header()` instead :)

Reply:
Nvm - I think I've misunderstood you design :)

Reply:
Yeah each runtime has modules which correspond to that particular chain's configuration. So the Millau chain has a file called `rialto.rs` where the configuration params for the Rialto chain are defined. If you think this is confusing I can put all that code is a common "config" folder or something. What do you think?

Reply:
No, it's perfectly fine :)